vignettes/methodological_introduction.Rmd

```diff
@@ -0,0 +1,109 @@
+
+Mixed models are called ''mixed'' because they consider both fixed and random effects and thus allow considerable modeling flexibility. This is the case even for the basic formulation described above. However, it still restricts within-group errors to be independent, identically distributed random variables with mean of zero and constant variance. These assumptions may often be seen as too restrictive (unrealistic) for applications. For example, in the case of clinical trials with  repeated measurements of subjects over time, observations are not independent and within-subject correlation needs to be accounted for by the model.
+
+
+## Extending the basic linear mixed-effects model
+
+The basic linear mixed-effects model can be extended in order to incorporate within-subject errors that are heteroscedastic (i.e.\ have unequal variances) and/ or are correlated.
+For this purpose, we can express the within-subject errors as:
+\[
+  \epsilon_i \sim \mathcal{N}(0,\sigma^2\Delta_i), \quad i=1,\ldots,M,
+\]
+where the $\Delta_i$ are positive-definite matrices parameterized by a fixed, generally small set of parameters <!-- $\lambda$-->. As in the basic model, the within-group errors $\epsilon_i$ are assumed to be independent for different $i$ and independent of the random effects $b_i$.
+
+The variance-covariance matrix of the response vector $y_i$,
+\[
+  \text{Var}(y_i) = \Sigma_i = \left( Z_i \Psi Z_{i}^{T} + \Delta_i \right),  
+\]
+comprises a random-effects component, given by $Z_i \Psi Z_{i}^{T}$, and a within-subject component, given by $\Delta_i$. 
+When fitting such models, there will generally be a "competition" and trade-off between the complexities of the two components, and care must be exercised to prevent nonidentifiability, or near nonidentifiability, of the parameters. This is one of the reasons why it can be advantageous in practice to use only one of these components as long as it still allows to capture all relevant sources of variability. In longitudinal studies with only one level of grouping, the within-subject component is particularly important to be considered in order to account for within-subject correlation, whereas an additional random-effects component is often not strictly needed.
+
+Of note, in the literature, the random-effects component and the within-subject component are sometimes also referred to as $R$ and $G$, or as $R$-side and $R$-side random effects (@Cnaan1997, @Littell2000).
+
+## The MMRM as a special case
+
+In a clinical trial setting, one often chooses to directly model the variance-covariance structure of the response, i.e. to account for within-subject dependencies using the within-group component $\Delta_i$, and can omit the random effects component ($Z_i b_i$). This yields the MMRM with:
+
+\[
+     y_i = X_i\beta + \epsilon_i, \quad \epsilon_i \sim \mathcal{N}(0,\sigma^2\Delta_i), \quad i=1,\ldots,M.
```

Comment:
would it be possible to adjust notation for the MMRM itself to be identical/very close to what we have in the other vignettes? in particular, see https://github.com/openpharma/mmrm/blob/main/vignettes/algorithm.Rmd#L52

Reply:
@danielinteractive : as requested, now updated for consistency with algorithm vignette